### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dayton1987**

### CODE REVIEW

**Comment by dayton1987** [comment]
[Widget & CKEditor: обновить](https://trello.com/c/alYu5WGU/152-widget-ckeditor-%D0%BE%D0%B1%D0%BD%D0%BE%D0%B2%D0%B8%D1%82%D1%8C)

File: README.md [hunk 1]
```diff
@@ -6,56 +6,72 @@
...
+  * [Automatic](#install-auto)
+  * [Manual](#install-manual)
+* [Usage](#usage)
+* [Configuration](#config)
+  * [Plugin config](#config-plugin)
+  * [Widget config](#config-widget)
+* [Feedback](#feedback)
 
-A minimalistic demo can be found [here][demo].
+## <a id="demo" href="#demo"></a> Demo
```
Comment (Zmoki) [inline README.md]: Its no needed, cause github create anchor links automatically

File: README.md [hunk 2]
```diff
@@ -65,37 +81,48 @@ CKEDITOR.replace('editor', {
...
 CKEDITOR.replace('editor', {
   extraPlugins: 'uploadcare',
   toolbar: [['Uploadcare', /* Your toolbar items */]],
   uploadcare: {
-    publicKey: 'YOUR_PUBLIC_KEY',
-    multiple: true,
-    crop: '1:1,4:3',
+    publicKey: 'YOUR_PUBLIC_KEY', // set your public API key here
+    multiple: true, // allow multi-file uploads
+    crop: '1:1,4:3', // set crop options when handling images
```
Comment (Zmoki) [inline README.md]: Also, users can add more "object key" options. Can we say about it here in `/* comment */` or after this code?

File: README.md (hunk 1 above)
Comment (Zmoki) [reply README.md]: Reveert all anchor links

File: README.md [hunk 3]
```diff
@@ -67,35 +85,66 @@ CKEDITOR.replace('editor', {
...
+
+## Contributors
+
+* [grayhound](https://github.com/grayhound)
+* [Zmoki](https://github.com/Zmoki)
+* [@dmitry-mukhin](https://github.com/dmitry-mukhin)
+* [nd0ut](https://github.com/nd0ut)
+* [homm](https://github.com/homm)
+* [disolovyov](https://github.com/disolovyov)
+* [zeroasterisk](https://github.com/zeroasterisk)
```
Comment (Zmoki) [inline README.md]: Maybe add link to github https://github.com/uploadcare/uploadcare-ckeditor/graphs/contributors ?

File: README.md [hunk 4]
```diff
@@ -107,3 +156,4 @@ Your feedback or support requests are welcome at [hello@uploadcare.com][uc-email
 [badge-stack-url]: https://stackshare.io/uploadcare/stacks/
 [badge-release-img]: https://img.shields.io/github/release/uploadcare/uploadcare-ckeditor.svg
 [badge-release-url]: https://github.com/uploadcare/uploadcare-ckeditor/releases
+[bounty]: mailto:bugbounty@uploadcare.com
```
Comment (Zmoki) [inline README.md]: `uc-email-bounty`. like `uc-email-hello`

File: README.md [hunk 5]
```diff
@@ -67,35 +85,66 @@ CKEDITOR.replace('editor', {
...
-  toolbar: [['Uploadcare', /* Your toolbar items */]],
+  toolbar: [['Uploadcare', /* your toolbar items */]],
   uploadcare: {
-    publicKey: 'YOUR_PUBLIC_KEY',
-    multiple: true,
-    crop: '1:1,4:3',
+    publicKey: 'YOUR_PUBLIC_KEY', // set your public API key here
+    multiple: true, // allow multi-file uploads
+    crop: '1:1,4:3', // set crop options when handling images
+    /* feel free to add more “object key” options here*/
```
Comment (Zmoki) [inline README.md]: space before `*/`

File: README.md [hunk 6]
```diff
@@ -67,35 +85,65 @@ CKEDITOR.replace('editor', {
...
 ```
 
-You can deeply customize the widget behavior: file sources, file validation, and much more.
-Please, check out the [Uploadcare Widget][widget-docs-config] and [JavaScript API][widget-docs-js-api] docs.
+### Widget configuration
+
+Uploadcare Widget can be deeply customized to suit your UX/UI. You can define
+allowed upload sources, implement file validation, and more.
+
+Use our live [widget sandbox][widget-configure] as a starting point and consider
```
[the remark this answers is not in the document]
Comment (Zmoki) [reply README.md]: also change link to `uc-widget-configure`

File: README.md [hunk 7]
```diff
@@ -27,7 +27,7 @@ with [Uploadcare Widget][uc-feature-widget].
 
 ## Demo
 
-Check out the basic demo [here][demo].
+Check out the basic demo [here][ck-demo].
```
Comment (Zmoki) [inline README.md]: Nope, demo is `demo`. Revert it

File: README.md [hunk 8]
```diff
@@ -49,8 +49,8 @@ You can find more info in the [CKBuilder docs][ck-docs-auto-install].
 
 ### Manual install
 
-Download the latest plugin archive from the [release branch][release-branch]
-or [releases page][releases-page].
+Download the latest plugin archive from the
+[release branch][github-release-branch] or [releases page][github-release-page].
```
Comment (Zmoki) [inline README.md]: `github-branch-release` & `github-releases`

File: README.md [hunk 9]
```diff
@@ -6,56 +6,75 @@
...
 
 CKEditor 4.0+ (not 5).
 
-A File Uploader for CKEditor v5 will be ready soon. Stay tuned.
+File Uploader for CKEditor v5 is on its way. Stay tuned.
 
 ## Install
 
-You’re free to choose from the install methods listed below.
+You can go with either [automatic](#install-auto) or [manual](install-manual)
```
Comment (Zmoki) [inline README.md]: wrong links

File: README.md [hunk 10]
```diff
@@ -67,43 +86,67 @@ CKEDITOR.replace('editor', {
...
 [badge-release-url]: https://github.com/uploadcare/uploadcare-ckeditor/releases
+[badge-stack-img]: https://img.shields.io/badge/tech-stack-0690fa.svg?style=flat
+[badge-stack-url]: https://stackshare.io/uploadcare/stacks/
+[demo]: https://uploadcare.github.io/uploadcare-ckeditor/
+[ck-docs-online-builder]: https://ckeditor.com/cke4/builder
+[ck-docs-auto-install]: https://docs.ckeditor.com/ckeditor4/latest/guide/dev_plugins.html#online-builder-installation
+[github-releases]: https://github.com/uploadcare/uploadcare-ckeditor/releases
+[github-branch-release]: https://github.com/uploadcare/uploadcare-ckeditor/tree/release
+[ck-docs-manual-install]: https://docs.ckeditor.com/ckeditor4/latest/guide/dev_plugins.html#manual-installation
+[uc-widget-docs-option-public-key]: https://uploadcare.com/docs/uploads/widget/config/#option-public-key/?utm_source=github&utm_campaign=uploadcare-ckeditor
```
Comment (Zmoki) [inline README.md]: ttps://uploadcare.com/docs/uploads/widget/config/?utm_source=github&utm_campaign=uploadcare-ckeditor#option-public-key

File: README.md [hunk 11]
```diff
@@ -67,43 +86,67 @@ CKEDITOR.replace('editor', {
...
+[uc-projects]: https://uploadcare.com/docs/keys/#projects/?utm_source=github&utm_campaign=uploadcare-ckeditor
```
Comment (Zmoki) [inline README.md]: https://uploadcare.com/docs/keys/?utm_source=github&utm_campaign=uploadcare-ckeditor#projects

**Pull request merged**